ptxboa/static/__init__.py

```diff
@@ -1,807 +1,807 @@
-    "LOHC (AEL)",
-    "LOHC (PEM)",
-    "LOHC (SOEC)",
-    "Methane (AEL)",
-    "Methane (AEL) + reconv. to H2",
-    "Methane (PEM)",
-    "Methane (PEM) + reconv. to H2",
-    "Methane (SOEC)",
-    "Methane (SOEC) + reconv. to H2",
-    "Methanol (AEL)",
-    "Methanol (PEM)",
-    "Methanol (SOEC)",
-]
-
-
-ScenarioType = Literal[
-    "2030 (high)",
-    "2030 (low)",
-    "2030 (medium)",
-    "2040 (high)",
-    "2040 (low)",
-    "2040 (medium)",
-]
-ScenarioValues = [
-    "2030 (high)",
-    "2030 (low)",
-    "2030 (medium)",
-    "2040 (high)",
-    "2040 (low)",
-    "2040 (medium)",
-]
-
-
-ResultClassType = Literal[
-    "Carbon",
-    "Derivate production",
-    "Electricity and H2 storage",
-    "Electricity generation",
-    "Electrolysis",
-    "HEAT",
-    "Transportation (Pipeline)",
-    "Transportation (Ship)",
-    "Water",
-]
-ResultClassValues = [
-    "Carbon",
-    "Derivate production",
-    "Electricity and H2 storage",
-    "Electricity generation",
-    "Electrolysis",
-    "HEAT",
-    "Transportation (Pipeline)",
-    "Transportation (Ship)",
-    "Water",
-]
-
-
-SecProcCO2Type = Literal["Direct Air Capture"]
-SecProcCO2Values = ["Direct Air Capture"]
-
-
-SecProcH2OType = Literal["Sea Water desalination"]
-SecProcH2OValues = ["Sea Water desalination"]
-
-
-ResGenType = Literal[
-    "PV tilted", "PV tracking", "Wind-PV-Hybrid", "Wind Offshore", "Wind Onshore"
-]
-ResGenValues = [
-    "PV tilted",
-    "PV tracking",
-    "Wind-PV-Hybrid",
-    "Wind Offshore",
-    "Wind Onshore",
-]
-
-
-ProcessCodeResType = Literal["PV-FIX", "PV-TRK", "RES-HYBR", "WIND-OFF", "WIND-ON"]
-ProcessCodeResValues = ["PV-FIX", "PV-TRK", "RES-HYBR", "WIND-OFF", "WIND-ON"]
-
-
-DimensionType = Literal[
-    "scenario",
-    "chain",
-    "region",
-    "country",
-    "transport",
-    "output_unit",
-    "flow",
-    "process",
-    "secproc_co2",
-    "secproc_water",
-    "res_gen",
-]
-DimensionValues = [
-    "scenario",
-    "chain",
-    "region",
-    "country",
-    "transport",
-    "output_unit",
-    "flow",
-    "process",
-    "secproc_co2",
-    "secproc_water",
-    "res_gen",
-]
-
-
-ProcessStepType = Literal[
-    "EL_STR",
-    "ELY",
-    "H2_STR",
-    "DERIV",
-    "PRE_SHP",
-    "PRE_PPL",
-    "POST_SHP",
-    "POST_PPL",
-    "SHP",
-    "SHP-OWN",
-    "PPLS",
-    "PPL",
-    "PPLX",
-    "PPLR",
-]
-ProcessStepValues = [
-    "EL_STR",
-    "ELY",
-    "H2_STR",
-    "DERIV",
-    "PRE_SHP",
-    "PRE_PPL",
-    "POST_SHP",
-    "POST_PPL",
-    "SHP",
-    "SHP-OWN",
-    "PPLS",
-    "PPL",
-    "PPLX",
-    "PPLR",
-]
-
-
-ResultCostType = Literal["CAPEX", "OPEX", "FLOW", "LC"]
-ResultCostValues = ["CAPEX", "OPEX", "FLOW", "LC"]
-
-
-TransportType = Literal["Ship", "Pipeline"]
-TransportValues = ["Ship", "Pipeline"]
-
-
-OutputUnitType = Literal["USD/MWh", "USD/t"]
-OutputUnitValues = ["USD/MWh", "USD/t"]
+# -*- coding: utf-8 -*-
+"""DO NOT EDIT (created by _update_static.py)."""

```

Comment:
Did you just replace "interest rate" with "WACC" in this file, or did you change it in the database as well?

Reply:
it is updated in the database as well as in the excel file which feeds to the database

Reply:
nice!